vita3k/renderer/include/renderer/vulkan/functions.h

```diff
@@ -24,7 +24,7 @@ struct MemState;
 
 namespace renderer::vulkan {
 
-bool create(SDL_Window *window, std::unique_ptr<renderer::State> &state, const char *base_path);
+bool create(SDL_Window *window, std::unique_ptr<renderer::State> &state, const char *base_path, const bool enable_validation_layer);
```

Comment:
Pass config to the create function instead of doing this

Reply:
done